test/proposals/zips/zip000.sol

```diff
@@ -264,6 +268,50 @@ contract zip000 is Proposal, TimelockProposal {
 
             addresses.addAddress("GAME_CONSUMABLE", address(consumer));
         }
+
+        {
+            Core core = Core(addresses.getAddress("CORE"));
+
+            // SeasonsTokenIdRegistry setup
+            SeasonsTokenIdRegistry seasonsTokenIdRegistry = new SeasonsTokenIdRegistry(address(core));
+            addresses.addAddress("SEASONS_TOKENID_REGISTRY", address(seasonsTokenIdRegistry));
+
+            /// ERC1155MaxSupplyMintable
+            string memory _metadataBaseUri = string(
+                abi.encodePacked("https://meta.", vm.envString("ENVIRONMENT"), vm.envString("DOMAIN"), "/")
+            );
+
+            // CapsulesNFT ERC1155 setup
+            ERC1155MaxSupplyMintable erc1155CapsulesNFT = new ERC1155MaxSupplyMintable(
+                address(core),
+                string(abi.encodePacked(_metadataBaseUri, "/seasons/1/capsules/metadata/")), //TODO confirm path
```

Comment:
The current consumables metadata URI is:

`/consumables/metadata/` 

What are your thoughts about changing what you have here to:

`/consumables/metadata/seasons/1/capsules/`

?

Reply:
fixed